src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -68,10 +68,16 @@ const ContentHighlightStepper = ({ enterpriseId }) => {
   const closeStepperModal = useCallback(() => {
     if (isCloseAlertOpen) {
       closeCloseAlert();
+      const trackInfo = {};
+      sendEnterpriseTrackEvent(
+        enterpriseId,
+        `${EVENT_NAMES.CONTENT_HIGHLIGHTS.STEPPER_CLOSE_STEPPER_INCOMPLETE}`,
+        trackInfo,
```

Comment:
nit: If there are no properties to send, you don't have to define `trackInfo` or pass it to `sendEnterpriseTrackEvent`